packages/theme-data/src/index.js

```diff
@@ -1,4 +1,6 @@
 export { default as basics } from "./basics";
+/* eslint-disable-next-line */
```

Comment:
disabled this test as I keep getting an `No named exports found in module` error when linting, open to suggestions on a better way to do this.

Reply:
One small thing, rather than use `eslint-disable-next-line` please use `eslint-disable-line whatever-rule-youre-disabling`. It makes it more clear what were disabling.